### DISCUSSION TITLE
Bind dashboard variables to Explore view when panel Explore link is used

### DISCUSSION THREAD

**Discussion by gjemp** [grafana/grafana#45673] [Observability]
**What would you like to be added**:
When I click Explore link in Dashboard panel header then I would be happy to have Dashboard Variables on the Explore view.

**Why is this needed**:
Today, it is crashing my browser and the explore UI is not meant for long list of values that is behind the variable in the source Dashboard. 
Also it would give the same UX feeling with the original panel  where you can switch same variables.

**Comment by aocenas** [comment]
@gjemp regarding the crash can you share which datasource it is?

for the feature request, do you mean you would like the variable controls which are present in dashboard to be also present in explore?

**Comment by gjemp** [comment]
it is influxdb

**Comment by Elfo404** [comment]
@gjemp  Not sure I got this correctly, are you asking to add the variables pickers in the explore view as well?

**Comment by gjemp** [comment]
I would assume that when i get to the explorer view from dashboard then it will clone ( ? )  dashboard variables for the query to explorer view also. How the UX  should look in that case , no good option for now

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
